usr/share/glib-2.0/schemas/org.x.sticky.gschema.xml

```diff
@@ -49,6 +49,14 @@
       </description>
     </key>
 
+    <key name='cycle-colors' type='b'>
+      <default>false</default>
+      <summary>Cycle colors</summary>
+      <description>
+        If true, each new not will be a differnt color from the last.
```

Comment:
Small typo in here `differnt`

Reply:
Fixed. Thanks for reviewing the code and letting me know :).